js/validation.js

```diff
@@ -17,3 +19,21 @@ form.addEventListener('submit', (event) => {
     form.submit();
   }
 });
+
+const button = document.querySelector('form .button');
+button.addEventListener('click', () => {
+  const formData = {
+    Name: nameInput.value,
+    Email: email.value,
+    Message: message.value,
+  };
+  localStorage.setItem('data', JSON.stringify(formData));
+});
+let storedData = localStorage.getItem('data'); storedData = JSON.parse(storedData);
+window.onload = () => {
+  if (storedData) {
+    nameInput.value = storedData.Name;
+    email.value = storedData.Email;
+    message.value = storedData.Message;
+  }
+};
```

Comment:
- [ ] Please note that on page reload the information entered in the contact form and not submitted should be retained and not lost 😎😎. Kindly make sure you look into this😉

Reply:
Please I have tried and tested several times and it is working. My partner has also tried it and it is working fine.
If it's still not working, kindly try switching to the local-storage branch in your code editor. Thanks